docs/cudf/source/conf.py

```diff
@@ -235,10 +237,14 @@ def resolve_aliases(app, doctree):
 
 def ignore_internal_references(app, env, node, contnode):
     name = node.get("reftarget", None)
-    if name is not None and name in _internal_names_to_ignore:
+    if name == "cudf.core.index.GenericIndex":
```

Comment:
Could you add a comment explaining why this is necessary? The internals of Sphinx node modifications are pretty arcane and won't be familiar to most people.

Reply:
Done